src/theory/strings/word.h

```diff
@@ -29,50 +30,53 @@ namespace strings {
 class Word
 {
 public:
-  /** make empty constant of kind k */
-  static Node mkEmptyWord(Kind k);
+ /** make empty constant of type tn */
```

Comment:
It looks like the formatting somehow got messed up (`public` is not indented at all and the `static` method only by one space. Can you ClangFormat the whole file?